src/Extend/Frontend.php

```diff
@@ -236,6 +272,24 @@ function (ActualFrontend $frontend, Container $container) {
         );
     }
 
+    private function registerPreloads(Container $container): void
+    {
+        if (empty($this->preloads)) {
+            return;
+        }
+
+        $container->resolving(
+            "flarum.frontend.$this->frontend",
+            function (ActualFrontend $frontend, Container $container) {
+                $frontend->content(ContainerUtil::wrapCallback(function (Document $document) {
+                    foreach ($this->preloads as $preload) {
+                        $document->preloads[] = is_callable($preload) ? $preload($document) : $preload;
```

Comment:
Am I right in thinking that wrapping with `wrapCallback` isn't needed here as it's already inside one of these?

Reply:
The content callback arg doesn't need to be wrapped in it. `wrapCallback` makes sure invokable classes work and are able to use dependency injection, it makes no difference to closures (callback functions) so it's not needed there.

Reply:
Gotcha. That should be right now :P